decoder/pokestop.go

```diff
@@ -638,12 +644,23 @@ func (stop *Pokestop) updatePokestopFromGetPokemonSizeContestEntryOutProto(conte
 			break
 		}
 		j.ContestEntries = append(j.ContestEntries, contestEntry{
-			Rank:      int(rank),
-			Score:     entry.GetScore(),
-			PokemonId: int(entry.GetPokedexId()),
-			Form:      int(entry.GetPokemonDisplay().Form),
-			Costume:   int(entry.GetPokemonDisplay().Costume),
-			Gender:    int(entry.GetPokemonDisplay().Gender),
+			Rank:                  int(rank),
+			Score:                 entry.GetScore(),
+			PokemonId:             int(entry.GetPokedexId()),
+			Form:                  int(entry.GetPokemonDisplay().Form),
+			Costume:               int(entry.GetPokemonDisplay().Costume),
+			Gender:                int(entry.GetPokemonDisplay().Gender),
+			Shiny:                 entry.GetPokemonDisplay().Shiny,
+			TempEvolution:         int(entry.GetPokemonDisplay().CurrentTempEvolution),
+			TempEvolutionFinishMs: entry.GetPokemonDisplay().TemporaryEvolutionFinishMs,
```

Comment:
this might be ns, not ms.... had the same issue on processing power stations... nia is not consistent with it.

Reply:
Maybe we can check the proto dump to confirm? :eyes:

Reply:
Regardless I'm in favor of keeping proto as is without doing any unit conversions. Those extra data might be useful for reverse engineering or something. We are not saving any storage/bandwidth anyways.